docs/source/transaction_family_specifications/xo_transaction_family.rst

```diff
@@ -0,0 +1,147 @@
+***********************************
+XO Transaction Family Specification
+***********************************
+
+Overview
+========
+
+The XO transaction family allows users to play a simple board game
+known variously as *tic-tac-toe*, *noughts and crosses*, and *XO*.
+
+State
+=====
+
+An XO state entry consists of the UTF-8 encoding of a string with
+exactly four commas formatted as follows:
+
+``<name>,<board>,<game-state>,<key1>,<key2>``
+
+where
+- <name> is a nonempty string not containing `|`,
+- <board> is a string of length nine containing only `O`, `X`, or `-`
+  (where the first three characters represent the top row of an XO
+  board, the next three represent the middle row, and the last three
+  represent the bottom row), and
+- <game-state> is one of the following: `P1-NEXT`, `P2-NEXT`,
+  `P1-WIN`, `P2-WIN`, or `TIE`.
+
+In the event of a hash collision (i.e. two or more state entries sharing the
+same address), the colliding state entries will stored as the UTF-8
+encoding of the string ``<a-entry>|<b-entry>|...``, where <a-entry>,
+<b-entry>,... are sorted alphabetically.
+
+
+Addressing
+----------
+
+XO data is stored in state using addresses generated from the XO
+family name and the name of the game being stored. In particular, an
+XO address consists of the first six characters of the SHA-512 hash of
+the UTF-8 encoding of the string "xo" (which is "5b7349") plus the
+first 64 characters of the SHA-512 hash of the UTF-8 encoding of the
```

Comment:
Here you wrote out the number "six" and used a numeral for "64". Should stick to one format especially since these numbers are being compared.